### PR TITLE
Added labels to display game status

### PR DESCRIPTION
**Pull request by omichel**
This PR adds labels to display the current game status in animations, so that it is easier to understand when watching the animation:
- who is playing against who.
- what is the current coverage status.
- what is the current KO status.
- who is the winner.

### CODE REVIEW

File: worlds/wrestling.wbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#VRML_SIM R2023a utf8
+#VRML_SIM R2023b utf8
```
Comment (Jean-Eudes-le-retour) [inline worlds/wrestling.wbt:1]: Do we want to set the version to R2023b or should we set it to the last stable version?

File: controllers/referee/referee.py [hunk 2]
```diff
@@ -1,120 +1,133 @@
...
+                if seconds != s:
+                    seconds = s
+                    minutes = int(time / 60000)
+                    self.display_time(minutes, seconds)
+                box = [0] * 3
+                for i in range(2):
+                    position = self.robot[i].getPosition()
+                    if abs(position[0]) < 1 and abs(position[1]) < 1:  # inside the ring
+                        coverage = 0
+                        for j in range(3):
```
Comment (Jean-Eudes-le-retour) [inline controllers/referee/referee.py:80]: I forgot to correct this: currently, the coverage metric also takes the z axis into account. Maybe we should remove it?
```suggestion
                        for j in range(2):
```

File: controllers/referee/referee.py [hunk 3]
```diff
@@ -1,120 +1,133 @@
...
+            self.digit[j][self.current_digit[j]].setPosition(1000)  # far away, not visible
+        self.current_digit[0] = minutes
+        self.current_digit[1] = seconds // 10
+        self.current_digit[2] = seconds % 10
+        for j in range(3):
+            self.digit[j][self.current_digit[j]].setPosition(0)  # visible
+
+    def run(self, CI):
+        # Performance output used by automated CI script
+        game_duration = 3 * 60 * 1000  # a game lasts 3 minutes
```
Comment (Jean-Eudes-le-retour) [inline controllers/referee/referee.py:56]: In the [README file](https://github.com/cyberbotics/wrestling/blob/main/README.md?plain=1#L92) we say that a match is 5 minutes long. I don't know if we want to increase the time limit in the code or if we want to change the README to be the same as the code; 3 minutes seems enough for me.

**Review by Jean-Eudes-le-retour — CHANGES_REQUESTED** [review]
Looks good ! I have added some comments

File: controllers/referee/referee.py (hunk 3 above)
Comment (omichel) [reply controllers/referee/referee.py:56]: 3 minutes looks for me as well. I will modify the README accordingly.

File: controllers/referee/referee.py (hunk 2 above)
Comment (omichel) [reply controllers/referee/referee.py:80]: Yes, I didn't noticed.

File: worlds/wrestling.wbt (hunk 1 above)
Comment (omichel) [reply worlds/wrestling.wbt]: Well spotted!

File: worlds/.wrestling.wbproj [hunk 4]
```diff
@@ -1,14 +1,18 @@
 Webots Project File version R2023b
```
Comment (Jean-Eudes-le-retour) [inline worlds/.wrestling.wbproj:1]: I think this is the last place where we need to change it.

**Review by Jean-Eudes-le-retour — CHANGES_REQUESTED** [review]
I've just noticed there is another mention to R2023b in .wrestling.wbproj

**Review by Jean-Eudes-le-retour — APPROVED** [review]
All good, cheers!

**Pull request merged**